ukmrapporter.php

```diff
@@ -119,10 +119,10 @@ public static function scripts_and_styles()
         wp_enqueue_style('WPbootstrap3_css');
         wp_enqueue_script('TwigJS');
 
-        wp_enqueue_style('UKMrapporter_css', self::getPluginUrl() . 'ukmrapporter.css');
-        wp_enqueue_script('UKMrapporter_js', self::getPluginUrl() . 'ukmrapporter.js');
+        wp_enqueue_style('UKMrapporter_css', static::getPluginUrl() . 'ukmrapporter.css');
+        wp_enqueue_script('UKMrapporter_js', static::getPluginUrl() . 'ukmrapporter.js');
 
-        wp_enqueue_style('jquery-ui-style', self::getPluginUrl() . 'UKMNorge/js/css/jquery-ui-1.7.3.custom.css');
+        wp_enqueue_style('jquery-ui-style', static::getPluginUrl() . 'UKMNorge/js/css/jquery-ui-1.7.3.custom.css');
         wp_enqueue_script('GOOGLEchart', 'https://www.google.com/jsapi');
```

Comment:
Huh, men dette funker jo ikke? Den fila finnes ikke 😂

Fjern parameter to i funksjonen, for jquery-ui-style registreres av [UKMNorge/UKMresources](https://github.com/UKMNorge/UKMresources) så da trenger vi ikke å definere url her